client/components/tinymce/plugins/simple-payments/dialog/list.jsx

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable wpcalypso/jsx-classname-namespace */
+
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import CompactCard from 'components/card/compact';
+
+class ProductList extends Component {
+	static propTypes = { paymentButtons: PropTypes.array.isRequired };
+
+	render() {
+		const { paymentButtons } = this.props;
+
+		return (
+			<div className="editor-simple-payments-modal__list">
+				{ paymentButtons.map( ( { ID: id, title, price, currency } ) => (
+					<CompactCard key={ id }>
+						<div>{ title }</div><div>{ price }&nbsp;{ currency }</div>
```

Comment:
According to our mockups, it should look like this:

![image](https://user-images.githubusercontent.com/4988512/28110460-cef252ae-66f2-11e7-9e2c-301579fdd6ba.png)

First is currency symbol and then price. You can use `lib/format-currency` to accomplish that easily.

Also, please don't use `&nbsp;`. :) If we need a space character, wrap the text into `span` or any other inline element.